Source/Engine/Platform/Base/NetworkBase.h

```diff
@@ -0,0 +1,255 @@
+// Copyright (c) 2012-2021 Wojciech Figat. All rights reserved.
+
+#pragma once
+
+#include "Engine/Core/Types/BaseTypes.h"
+#include "Engine/Core/Types/String.h"
+API_INJECT_CPP_CODE("#include \"Engine/Platform/Network.h\"");
+#define SOCKGROUP_MAXCOUNT 64
+#define SOCKGROUP_ITEMSIZE 16
+
+enum class FLAXENGINE_API NetworkProtocolType
+{
+    Undefined,
+    Udp,
+    Tcp
+};
+
+enum class FLAXENGINE_API NetworkIPVersion
+{
+    Undefined,
+    IPv4,
+    IPv6
+};
+
+struct FLAXENGINE_API NetworkSocket
+{
+    NetworkProtocolType Protocol = NetworkProtocolType::Undefined;
+    NetworkIPVersion IPVersion = NetworkIPVersion::Undefined;
+    byte Data[8] = {}; // sizeof SOCKET = unsigned __int64
+};
+
+struct FLAXENGINE_API NetworkEndPoint
+{
+    NetworkIPVersion IPVersion = NetworkIPVersion::Undefined;
+    String Address;
+    String Port;
+    byte Data[28] = {}; // sizeof sockaddr_in6 , biggest sockaddr that we will use
+};
+
+enum FLAXENGINE_API NetworkSocketOption
+{
+    Debug,
+    ReuseAddr,
+    KeepAlive,
+    DontRoute,
+    Broadcast,
+    UseLoopback,
+    Linger,
+    OOBInline,
+    SendBuffer,
+    RecvBuffer,
+    SendTimeout,
+    RecvTimeout,
+    Error,
+    NoDelay,
+    IPv6Only
+};
+
+struct FLAXENGINE_API NetworkSocketState
+{
+    bool Error = false;
+    bool Invalid = false;
+    bool Disconnected = false;
+    bool Readable = false;
+    bool Writeable = false;
+};
+
+struct FLAXENGINE_API NetworkSocketGroup
+{
+    uint32 Count;
+    byte Data[SOCKGROUP_MAXCOUNT * 16] = {};
+};
+
+class FLAXENGINE_API NetworkBase
+{
+    public:
+    /// <summary>
+    /// Initialize the network module.
+    /// </summary>
+    /// <returns>Return true on error. Otherwise false.</returns>
+    static bool Init();
```

Comment:
Change doc comments form from `Initialize` to `Initializes` (and so on for other methods, `Close` -> `Closes`).